### PR TITLE
Docs: Document Promtail global rate limiting

### PR DESCRIPTION
**Pull request by KMiller-Grafana**
Addresses issue https://github.com/grafana/loki/issues/5730.

Reviewers: what have I missed or gotten wrong in this addition of documentation?

### CODE REVIEW

File: docs/sources/clients/promtail/configuration.md [hunk 1]
```diff
@@ -1756,6 +1759,26 @@ scrape_configs:
...
+# The rate limit in log lines per second that this instance of Promtail may push to Loki.
+[readline_rate: <int> | default = 10000]
+
+# The cap in the quantity of burst lines that this instance of Promtail may push
+# to Loki.
+[readline_burst: <int> | default = 10000]
+
+# When true, exceeding the rate limit causes this instance of Promtail to discard
+# log lines, rather than sending them to Loki. When false, exceeding the rate limit
+# causes this instance of Promtail to hold off on sending the log lines.
```
Comment (09jvilla) [inline docs/sources/clients/promtail/configuration.md:1779]:
```suggestion
# causes this instance of Promtail to temporarily hold off on sending the log lines and retry later.
```

**Review by 09jvilla — CHANGES_REQUESTED** [review]
Made one small suggestion - realize its a tad vague because I don't actually know the logic for how long it holds the lines back.

File: docs/sources/clients/promtail/configuration.md [hunk 2]
```diff
@@ -1756,6 +1759,26 @@ scrape_configs:
         target_label: 'container'
 ```
 
+## limit_config
```
Comment (dannykopping) [inline docs/sources/clients/promtail/configuration.md:1762]: This looks good @KMiller-Grafana, and we should align this with the flags registered here:
https://github.com/grafana/loki/blob/main/clients/pkg/promtail/limit/config.go#L14-L19

**Review by dannykopping — APPROVED** [review]
LGTM! Congrats on your first code change Karen :)

File: docs/sources/clients/promtail/configuration.md [hunk 3]
```diff
@@ -95,6 +95,9 @@ clients:
 scrape_configs:
   - [<scrape_config>]
 
+# Configures global limits for this instance of Promtail
+[limit_config: <limit_config>]
```
Comment (liguozhong) [inline docs/sources/clients/promtail/configuration.md:99]: hi, thank you very much for the exquisite documentation, this PR changed the limit_config name

https://github.com/grafana/loki/pull/5707
Promtail: Rename config name limit_config to limits_config #5707

Comment (09jvilla) [inline docs/sources/clients/promtail/configuration.md:99]:
```suggestion
[limits_config: <limits_config>]
```

File: docs/sources/clients/promtail/configuration.md (hunk 2 above)
Comment (09jvilla) [inline docs/sources/clients/promtail/configuration.md:1762]:
```suggestion
## limits_config
```

File: docs/sources/clients/promtail/configuration.md [hunk 4]
```diff
@@ -1756,6 +1759,26 @@ scrape_configs:
         target_label: 'container'
 ```
 
+## limit_config
+
+The optional `limit_config` block configures global limits for this instance of Promtail.
```
Comment (09jvilla) [inline docs/sources/clients/promtail/configuration.md:1764]:
```suggestion
The optional `limits_config` block configures global limits for this instance of Promtail.
```

**Review by 09jvilla — CHANGES_REQUESTED** [review]
Looks good to me - I think all that's needed is making "limits" plural (revise from "limit")

**Review by 09jvilla — APPROVED** [review]
LGTM!

**Pull request merged**